README.md

```diff
@@ -33,12 +31,4 @@ This dashboard wouldn't have been made without the help and inspiration from the
 * [Jonah](https://github.com/jwbonner) from Team 6328
 * [Oh yes 10 FPS](https://github.com/oh-yes-0-fps) from Team 3173
 * [Jason](https://github.com/jasondaming) and [Peter](https://github.com/PeterJohnson) from WPILib
-* [MikLast](https://www.chiefdelphi.com/u/MikLast), FRC Alum, Mentor, & Volunteer
```

Comment:
These images are still included, right?

Reply:
Yes, but I added the sources to them within the app and gave credit to him for the images